### PR TITLE
Add bridged token adresses via remark plugin

### LINKED ISSUE #181: Add new page with bridged token addresses
### Description

Add a new page "Bridged token adresses", which lists all bridged tokens on Lisk analog to [Bridged Token Adresses](https://docs.optimism.io/chain/tokenlist).

Investigate opportunities to automate the process of updating the page, so the page will keep itself up-to-date

### Motivation

To provide an overview about tokens on Lisk.

### Additional Information
- https://github.com/ethereum-optimism/ethereum-optimism.github.io/tree/master/data
- https://typescript-eslint.io/blog/automated-rule-docs-with-docusaurus-and-remark/
- https://github.com/remarkjs/remark
- https://github.com/unifiedjs/unified
- https://github.com/syntax-tree/mdast

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #181

### CODE REVIEW

File: plugins/generated-bridged-token-adresses-docs.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                          type: 'text',
+                          value: token.symbol
+                        }]
+                    },
+                    {
+                      type: 'tableCell',
+                      children: [
+                        {
+                          type: 'link',
+                          url: "https://sepolia.etherscan.io/address/" + token.ethAddress+ ")",
```
Comment (matjazv) [inline plugins/generated-bridged-token-adresses-docs.js]: `+ ")"` makes link broken.

File: plugins/generated-bridged-token-adresses-docs.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                          children: [{type: 'text', value: token.ethAddress }]
+                        }
+                      ]
+                    },
+                    {
+                      type: 'tableCell',
+                      children: [
+                        {
+                          type: 'link',
+                          url: "https://sepolia-blockscout.lisk.com/address/" + token.address+ ")",
```
Comment (matjazv) [inline plugins/generated-bridged-token-adresses-docs.js]: `+ ")"` makes link broken.

**Review by matjazv — CHANGES_REQUESTED** [review]

**Review by matjazv — APPROVED** [review]
LGTM!

The only question I have is if we want to have Lisk token at the top of both tables or not?

**Comment by Tschakki** [comment]
> LGTM!
> 
> The only question I have is if we want to have Lisk token at the top of both tables or not?

It is currently on top because it is the only entry. I am not sure that it will be on top once more tokens are bridged to Lisk.

**Comment by matjazv** [comment]
> > LGTM!
> > The only question I have is if we want to have Lisk token at the top of both tables or not?
> 
> It is currently on top because it is the only entry. I am not sure that it will be on top once more tokens are bridged to Lisk.

Yes, I was asking for the future when there'll be more tokens listed.
Probably they will be listed as how they're listed inside `.json` file so Lisk token surely won't be at the top inside those two tables.
Not sure if we want that Lisk token is always at the top of the table or not...

**Comment by Tschakki** [comment]
> Not sure if we want that Lisk token is always at the top of the table or not...

I see... I think it is ok  like this for now. If the list gets longer, we can consider to just hardcode the Lisk addresses at the top of the table, and only add other tokens dynamically.

File: docusaurus.config.js [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
 import { themes as prismThemes } from 'prism-react-renderer';
 import * as process from 'process';
+import { generatedDocs } from './plugins/generated-bridged-token-adresses-docs';
```
Comment (TalhaMaliktz) [inline docusaurus.config.js:9]: What are these about?
Saw them as a result of `npm run build` and `npm run start`
image

File: plugins/generated-bridged-token-adresses-docs.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                          type: 'link',
+                          url: "https://blockscout.lisk.com/address/" + token.address,
+                          children: [{type: 'text', value: token.address }]
+                        }
+                      ]
+                    }]
+                  });
+                } else {
+                  console.log("Info: chainId not found")
+                }
```
Comment (TalhaMaliktz) [inline plugins/generated-bridged-token-adresses-docs.js]: This `else` is getting executed, which shouldn't happen if a table row is being pushed!

File: plugins/generated-bridged-token-adresses-docs.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+                        {
+                          type: 'link',
+                          url: "https://sepolia-blockscout.lisk.com/address/" + token.address,
+                          children: [{type: 'text', value: token.address }]
+                        }
+                      ]
+                    }]
+                  });
+                } else {
+                  console.log("Error: chainId not found")
```
Comment (TalhaMaliktz) [inline plugins/generated-bridged-token-adresses-docs.js]: This `else` is getting executed, which shouldn't happen if a table row is being pushed!

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

File: plugins/generated-bridged-token-adresses-docs.js (hunk 4 above)
Comment (Tschakki) [reply plugins/generated-bridged-token-adresses-docs.js]: log was in wrong place, removed it now

File: plugins/generated-bridged-token-adresses-docs.js (hunk 5 above)
Comment (Tschakki) [reply plugins/generated-bridged-token-adresses-docs.js]: log was in wrong place, removed it now

File: docusaurus.config.js (hunk 3 above)
Comment (Tschakki) [reply docusaurus.config.js:9]: I am not sure, but it looks like the plugin is imported anyway 🤔

Comment (TalhaMaliktz) [reply docusaurus.config.js:9]: That seems to be the case! however, let's work on debugging this to keep our repository clean. I've already tried removing `node_modules` and the `.lock` file, but no luck. It seems like there's something going on with the `config` and the `generatedDocs` file. Let's figure this out together!

**Review by TalhaMaliktz — APPROVED** [review]

**Pull request merged**